LICENCE.txt

```diff
@@ -0,0 +1,8 @@
+Akeneo PIM PHP Client
+
+The Open Software License version 3.0
+
+Copyright (c) 2013-2017, Akeneo SAS.
+
+Full license is at: http://opensource.org/licenses/OSL-3.0
```

Comment:
not sure about the license

Reply:
I asked to @nidup for the CE, maybe it can tell us, because I don't know too

Reply:
I asked we should use OSL-3.0